content/soarian/dstu2/medications/medication-statement.md

```diff
@@ -0,0 +1,83 @@
+---
+title: MedicationStatement | Soarian DSTU 2 API
```

Comment:
Modified binding table to have only contained binding and added comment about contained bindings in the implementation notes section